content/docs/command-reference/remote/add.md

```diff
@@ -115,21 +115,7 @@ $ dvc remote add -d myremote s3://mybucket/path
 For more on the supported env vars, please see the
 [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#environment-variable-configuration)
 
-The following API methods are performed by the `boto3` library (used by DVC):
```

Comment:
hmm .. why do we remove this?

Reply:
It's a bit too detailed (most deep details are in `remote modify`. I thought that's what you were referring to in https://github.com/iterative/dvc.org/pull/2415#pullrequestreview-650598131 haha. OK, no strong opinion here, will reinstate it ⌛

Reply:
these details are general, about this type of the remote, they are not about auth - that's what we should keep in mind. Eventually this kind of a spec will go into the UG (similar to the GDrive).

But for now this general remote content probably belong better here, not in the modify. Some details and language can be updated (or keep for now as is if it's tricky)

Reply:
Got it, makes sense.

I compacted those 2 bullet lists but kept them in both refs for now (dfdaf404), because `remote modify` has some params that refer to the API functions used.